internal/sdkprovider/service/kafka/kafka_native_acl_test.go

```diff
@@ -0,0 +1,74 @@
+package kafka_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	acc "github.com/aiven/terraform-provider-aiven/internal/acctest"
+)
+
+// TestKafkaNativeAcl tests the kafka acl resource.
+func TestKafkaNativeAcl(t *testing.T) {
+	projectName := os.Getenv("AIVEN_PROJECT_NAME")
+	serviceName := acc.RandStr()
```

Comment:
nit: let's use `serviceName := fmt.Sprintf("test-acc-sr-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))` for consistency with other tests

![image](https://github.com/user-attachments/assets/f171e569-9d2e-4e65-bb92-c2d913af61ce)

Reply:
I believe, adding some helping prefix will do the job: `"test-acc-native-acl-%s"`